### ISSUE TITLE
Django 1.10 TypeError at /api/docs/ argument of type 'NoneType' is not iterable

### ISSUE THREAD

**Issue by grillazz** [marcgibbons/django-rest-swagger#517]
Hi,

When I switch  to Django 1.10 and DRF 3.4.4 I can't use SwaggerUIRenderer anymore with error: TypeError at /api/docs/ argument of type 'NoneType' is not iterable.
CoreJSONRenderer and OpenAPIRenderer are working well.

Please find traceback below.

Thanks,

Jakub

```python
Environment:


Request Method: GET
Request URL: http://0.0.0.0:8000/api/docs/

Django Version: 1.10
Python Version: 3.5.2
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'rest_framework',
 'rest_framework_swagger',
 'api']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']



Traceback:

File "/usr/local/lib/python3.5/site-packages/django/shortcuts.py" in resolve_url
  147.         return reverse(to, args=args, kwargs=kwargs)

File "/usr/local/lib/python3.5/site-packages/django/urls/base.py" in reverse
  91.     return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))

File "/usr/local/lib/python3.5/site-packages/django/urls/resolvers.py" in _reverse_with_prefix
  389.             (lookup_view_s, args, kwargs, len(patterns), patterns)

During handling of the above exception (Reverse for 'None' with arguments '()' and keyword arguments '{}' not found. 0 pattern(s) tried: []), another exception occurred:

File "/usr/local/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
  39.             response = get_response(request)

File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  217.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  215.                 response = response.render()

File "/usr/local/lib/python3.5/site-packages/django/template/response.py" in render
  109.             self.content = self.rendered_content

File "/usr/local/lib/python3.5/site-packages/rest_framework/response.py" in rendered_content
  72.         ret = renderer.render(self.data, accepted_media_type, context)

File "/usr/local/lib/python3.5/site-packages/rest_framework_swagger/renderers.py" in render
  51.         self.set_context(renderer_context)

File "/usr/local/lib/python3.5/site-packages/rest_framework_swagger/renderers.py" in set_context
  61.         renderer_context.update(self.get_auth_urls(renderer_context))

File "/usr/local/lib/python3.5/site-packages/rest_framework_swagger/renderers.py" in get_auth_urls
  66.             for setting, url in self.get_auth_url_settings().items()

File "/usr/local/lib/python3.5/site-packages/rest_framework_swagger/renderers.py" in <dictcomp>
  66.             for setting, url in self.get_auth_url_settings().items()

File "/usr/local/lib/python3.5/site-packages/rest_framework_swagger/renderers.py" in add_next_to_url
  83.         return '%s?next=%s' % (resolve_url(url), request.path)

File "/usr/local/lib/python3.5/site-packages/django/shortcuts.py" in resolve_url
  153.         if '/' not in to and '.' not in to:

Exception Type: TypeError at /api/docs/
Exception Value: argument of type 'NoneType' is not iterable
```

**Comment by grillazz** [comment]
ps. when I shrink swagger renderer to below everything seems to fine with Django 1.10

```python
class SwaggerUIRenderer(BaseRenderer):
    media_type = 'text/html'
    format = 'swagger'
    template = 'rest_framework_swagger/index.html'
    charset = 'utf-8'

    def render(self, data, accepted_media_type=None, renderer_context=None):
        return render(
            renderer_context['request'],
            self.template,
            renderer_context
        )
```

**Comment by crearc** [comment]
I had to set my LOGIN_URL and LOGOUT_URL in SWAGGER_SETTINGS:

https://github.com/marcgibbons/django-rest-swagger/blob/c2ab269950a9d1f2c1e9833026df0251ba293e6e/docs/settings.md#login_url

**Comment by smarkwell** [comment]
Just encountered this issue, as I have a Django sample app I'm working on that does not have Authentication or a database model at all.  The above patch looks like it would fix the issue I'm running into as well.


```
SWAGGER_SETTINGS = {
    'USE_SESSION_AUTH': False,
    'LOGIN_URL': None,
    'LOGOUT_URL': None,
}
```

**Comment by marcgibbons** [comment]
Fixed in #520.
